mongodbatlas/data_source_mongodbatlas_cloud_backup_schedule.go

```diff
@@ -25,6 +25,37 @@ func dataSourceMongoDBAtlasCloudBackupSchedule() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+			"copy_settings": {
```

Comment:
@martinstibbe can you confirm that we don't need to make any updates to [go-client](https://github.com/mongodb/go-client-mongodb-atlas)?

Reply:
copy_settings is supported in go-client
`type CloudProviderSnapshotBackupPolicy struct {
	.....
	CopySettings                      []CopySetting        `json:"copySettings"`                                // List that contains a document for each copy setting item in the desired backup policy.
	DeleteCopiedBackups               []DeleteCopiedBackup `json:"deleteCopiedBackups,omitempty"`               // List that contains a document for each deleted copy setting whose backup copies you want to delete.
}
`

Reply:
sounds good!